can/zynqmp-can/zynqmp-can.c

```diff
@@ -0,0 +1,552 @@
+	uint32_t RXFIFO_DATA1;    /* Rx Message FIFO, data word 1., Address offset: 0x58 */
+	uint32_t RXFIFO_DATA2;    /* Rx Message FIFO, data word 2., Address offset: 0x5c */
+	uint32_t AFR;             /* Acceptance Filter Enables., Address offset: 0x60 */
+	uint32_t AFMR1;           /* Acceptance Filter 1 Mask., Address offset: 0x64 */
+	uint32_t AFIR1;           /* Acceptance Filter 1 ID., Address offset: 0x68 */
+	uint32_t AFMR2;           /* Acceptance Filter 2 Mask., Address offset: 0x6c */
+	uint32_t AFIR2;           /* Acceptance Filter 2 ID., Address offset: 0x70 */
+	uint32_t AFMR3;           /* Acceptance Filter 3 Mask., Address offset: 0x74 */
+	uint32_t AFIR3;           /* Acceptance Filter 3 ID., Address offset: 0x78 */
+	uint32_t AFMR4;           /* Acceptance Filter 4 Mask., Address offset: 0x7c */
+	uint32_t AFIR4;           /* Acceptance Filter 4 ID., Address offset: 0x80 */
+} can_periph_t;
+
+/* Data structure used by the driver */
+static struct {
+	volatile can_periph_t *base; /**< Pointer to the peripheral base address */
+	oid_t oid;                   /**< Object identifier for the related device file */
+	handle_t cond;               /**< Conditional variable for synchronizing interrupts */
+	handle_t inth;               /**< Interrupt handler object */
+	handle_t lock;               /**< Mutex used with the conditional variable for synchronization */
+} can = { 0 };
+
+/* Immutable information about CAN peripheral instances */
+static const struct {
+	uint32_t base;    /**< Peripheral base address */
+	unsigned int irq; /**< IRQ number */
+	uint16_t clk;     /**< Identifier of the related clock in CPU clock registers */
+	uint16_t rst;     /**< Identifier of the related reset line in the CPU reset peripheral */
+	uint16_t rxPin;   /**< CAN RX pin (configured in board_config.h) */
+	uint16_t txPin;   /**< CAN TX pin (configured in board_config.h) */
+} info[canInstancesCount] = {
+	{ ZYNQMP_CAN0_BASE, 55, pctl_devclock_lpd_can0, pctl_devreset_lpd_can0, CAN0_RX, CAN0_TX },
+	{ ZYNQMP_CAN1_BASE, 56, pctl_devclock_lpd_can1, pctl_devreset_lpd_can1, CAN1_RX, CAN1_TX }
+};
+
+/* Popular baud rate configurations */
+static const struct {
+	uint32_t baudrate_kbps; /**< Baud rate in kbps */
+	uint32_t brpr;          /**< Baud Rate Prescaler register value */
+	uint32_t btr;           /**< Bit Timing register value */
+	char *sample;           /**< Sampling point percentage */
+} timeCfg[] = {
+	{ .baudrate_kbps = 50, .sample = "87.5", .brpr = 0x7C, .btr = 0x05 },
+	{ .baudrate_kbps = 100, .sample = "90.0", .brpr = 0x31, .btr = 0x07 },
+	{ .baudrate_kbps = 125, .sample = "87.5", .brpr = 0x31, .btr = 0x05 },
+	{ .baudrate_kbps = 250, .sample = "87.5", .brpr = 0x18, .btr = 0x05 },
+	{ .baudrate_kbps = 500, .sample = "90.0", .brpr = 0x09, .btr = 0x07 },
+	{ .baudrate_kbps = 1000, .sample = "90.0", .brpr = 0x04, .btr = 0x07 },
+};
+
+/* Function to display command-line help */
+static void cli_help(const char *progname)
```

Comment:
All functions, even static ones, shall have relevant prefix. Use camelCase
```suggestion
static void zynqmpCan_cliHelp(const char *progname)
```